vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -916,6 +917,42 @@ void Publisher::SavePublisherInfo(
       });
 }
 
+// static
+std::string Publisher::GetShareURL(
```

Comment:
can you please add unit tests for this function

Reply:
Good idea, added.